app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -0,0 +1,23 @@
+module V0
```

Comment:
I think it makes more sense to have education/v0 than v0/education, since we'll have multiple APIs in one repo and in the future they may be out of sync with some on v1 permanently and others moving up to v3 or whatever.